apps/docs/docs/react/install.mdx

```diff
@@ -0,0 +1,27 @@
+---
+sidebar_position: 1
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Install
+
+By default Griffel contains only runtime version, it's enough to install it and it works:
```

Comment:
```suggestion
By default Griffel is a runtime css-in-js engine, you can simply install and use it in code directly:
```

This might be worded a bit clearer ?